src/dashboard/apps/consent/tests/test_models.py

```diff
@@ -67,3 +121,16 @@ def test_update_consent_status():
     assert consent.status == REVOKED
     assert consent.updated_at > new_updated_at
     assert consent.revoked_at is not None
+
+    # update start and end date
+    expected_start_date = make_aware(
+        datetime.datetime.strptime("2024-12-20", "%Y-%m-%d")
+    )
+    expected_end_date = make_aware(datetime.datetime.strptime("2024-12-22", "%Y-%m-%d"))
+
+    consent.start = expected_start_date
+    consent.end = expected_end_date
+    consent.save()
+
+    assert consent.start == expected_start_date
+    assert consent.end == expected_end_date
```

Comment:
I don't get the purpose of those assertions.

Reply:
it is true that they have no interest